docs/src/pages/components/data-grid/demo/FullFeaturedDemo.js

```diff
@@ -148,14 +148,14 @@ function SettingsPanel(props) {
 
   return (
     <FormGroup className="MuiFormGroup-options" row>
-      <FormControl>
+      <FormControl variant="standard">
```

Comment:
it seems like it should be the default value.

Reply:
Standard was the default value, we changed it to outlined. I could gave gone the opposite direction, make the input field outlined but it seems to ues more space, so I didn't.

Reply:
I think it is better as it is/was.

 I meant that standard as a name for me should be the default. 
Here, the default is outlined, and the standard is not the default

Reply:
Yes, standard is no longer the default in v5. As a matter of fact, standard is no longer inside the material design guidelines since 2018-2019.